bitnami/moodle/values.yaml

```diff
@@ -81,6 +81,9 @@ moodleSkipInstall: false
 ## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
 ##
 moodleSiteName: ""
+## @param moodleLang Site language
+## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
+moodleLang: ""
```

Comment:
```suggestion
## @param moodleLang Site language
## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
##
moodleLang: ""
```

Reply:
indeed I have just seen that the chat is already in version 14.2.6. I will bump it in 14.2.7